shell.nix

```diff
@@ -24,24 +24,24 @@ let
 
     # These programs will be available inside the nix-shell.
     buildInputs = with haskellPackages; [
-      #cardanoDbSync.cardano-db-sync
-      #cardanoNode.cardano-cli
-      #cardanoNode.cardano-node
-      #cabal-install
-      #stack
-      #ghcid
-      #hlint
+      cardanoDbSync.cardano-db-sync
```

Comment:
I would prefer if you do not readd these dependencies here.

Reply:
@CodiePP, we can do away with `cardano-db-sync`, since we don't really use it.

Which other ones are problematic (and why)?

Reply:
Dropped all but the necessary packages.

Will resume the conversation with devops about cache divergence.